packages/dapp/src/apis/civilTCR.ts

```diff
@@ -116,11 +116,36 @@ export async function getNewsroom(address: EthAddress): Promise<any> {
   return newsroom;
 }
 
-export async function getParameterValue(param: string): Promise<BigNumber> {
+export async function getParameterValue(params: string[]): Promise<BigNumber[]> {
```

Comment:
Should we change this method name to `multiGetParameterValue` or `getParameterValues` something like that so it's clear when used in code? Or alternatively, keep `getParmeterValue` as a getter for a single param value and add another method called `multiGetParmeterValue`?